pkgs/development/python-modules/hypothesis/default.nix

```
@@ -79,6 +79,14 @@ buildPythonPackage rec {
     "tests/cover"
   ];
 
+  disabledTests = lib.optionals (pythonOlder "3.10") [
+    # not sure why these tests fail with only 3.9
+    # FileNotFoundError: [Errno 2] No such file or directory: 'git'
+    "test_observability"
+    "test_assume_has_status_reason"
+    "test_observability_captures_stateful_reprs"
+  ];
```

```suggestion
  disabledTests = if (pythonOlder "3.10") then [
    # not sure why these tests fail with only 3.9
    # FileNotFoundError: [Errno 2] No such file or directory: 'git'
    "test_observability"
    "test_assume_has_status_reason"
    "test_observability_captures_stateful_reprs"
  ] else null;
```

Or you'll cause a mass-rebuild.

```
nix-repl> lib.optionals false [ "foo" ] 
[ ]
```

---

I'm not interested in mass-rebuild. Thanks.